tests/Functional/Bundle/Maker/MakeFactoryTest.php

```diff
@@ -298,12 +302,12 @@ public function can_customize_namespace_with_test_flag_with_root_namespace_prefi
     /**
      * @test
      */
-    public function can_create_factory_with_all_interactively(): void
+    public function can_create_all_factories_for_doctrine_objects(): void
     {
         $tester = new CommandTester((new Application(self::bootKernel()))->find('make:factory'));
 
         $this->assertFileDoesNotExist(self::tempFile('src/Factory/CategoryFactory.php'));
-        $this->assertFileDoesNotExist(self::tempFile('src/Factory/PostFactory.php'));
+        $this->assertFileDoesNotExist(self::tempFile('src/Factory/TagFactory.php'));
```

Comment:
we have a problem when different factories will have the same name: `ODM/Category.php` and `ORM/Category.php` will reasult in the same `Factory/CategoryFactory.php` file, then the second one will create an error. 

I've updated https://github.com/zenstruck/foundry/issues/336 to handle this case